### PR TITLE
Update audio-conferencing-subscription-dial-out.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Adding a few additional details to the article

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ ms.custom:
...
 
-Customers have been provided a [complimentary dial-out capability](complimentary-dial-out-period.md) that will end on November 30, 2019. Beginning December 1, 2019, each audio conferencing subscription provides 60 minutes per user per month to ANY of the Zone A countries as described in this document. Your tenant dial-out minute pool size is based on *assigned* licenses and not purchased licenses. This benefit is applicable to Audio Conferencing *monthly subscription* licenses and does not extend to Audio Conferencing pay-per-minute licenses. 
+Customers have been provided a [complimentary dial-out capability](complimentary-dial-out-period.md) that will end on November 30, 2019. Beginning December 1, 2019, each Audio Conferencing subscription provides 60 minutes per user per month that can be used to dial out to non-premium numbers in ANY of the Zone A countries as described in this document. Your tenant dial-out minute pool size is based on *assigned* licenses and not purchased licenses. This benefit is applicable to Audio Conferencing *monthly subscription* licenses and does not extend to Audio Conferencing pay-per-minute licenses.
```
Comment (nenonix) [inline Teams/audio-conferencing-subscription-dial-out.md]:
```suggestion
Customers have been provided a [complimentary dial-out capability](complimentary-dial-out-period.md) that will end on November 30, 2019. Beginning December 1, 2019, each Audio Conferencing subscription provides 60 minutes per user per month. These minutes can be used to dial out to non-premium numbers in ANY of the Zone A countries as described in this document. Your tenant dial-out minute pool size is based on *assigned* licenses and not purchased licenses. This benefit is applicable to Audio Conferencing *monthly subscription* licenses and does not extend to Audio Conferencing pay-per-minute licenses. 
```

File: Teams/audio-conferencing-subscription-dial-out.md [hunk 2]
```diff
@@ -52,10 +52,10 @@ For customers adopting our Audio Conferencing service, Microsoft provides the ab
...
-- All outbound calls to destinations not in the Zone A country list are billed per minute using Communications Credits at our published rates to that destination (provided the customer has set up Communications Credits and assigned the Communications Credits license to the meeting organizer).
+- All dial-out calls exceeding 6,900 minutes per calendar month are billed per minute using Communications Credits at our published rates to that destination. (Note: The customer must set up [Communications Credits](what-are-communications-credits.md) and assign the Communications Credits license to the meeting organizer.)
+- All dial-out calls to destinations not in the Zone A country list are billed per minute using Communications Credits at our published rates to that destination (provided the customer has set up Communications Credits and assigned the Communications Credits license to the meeting organizer).
```
Comment (nenonix) [inline Teams/audio-conferencing-subscription-dial-out.md]:
```suggestion
- All dial-out calls to destinations that are not in the Zone A country list are billed per minute using Communications Credits at our published rates to that destination (provided the customer has set up Communications Credits and assigned the Communications Credits license to the meeting organizer).
```

File: Teams/audio-conferencing-subscription-dial-out.md (hunk 1 above)
Comment (ruizoscar) [reply Teams/audio-conferencing-subscription-dial-out.md]: The first sentences on this paragraph were constructed with CELA, could we please keep the original change?

Comment (nenonix) [reply Teams/audio-conferencing-subscription-dial-out.md]: Absolutely, copy edit suggestions are just suggestions and should never get in the way of acccurate technical information

File: Teams/audio-conferencing-subscription-dial-out.md (hunk 2 above)
Comment (ruizoscar) [reply Teams/audio-conferencing-subscription-dial-out.md]: This change looks good, thanks!

File: Teams/audio-conferencing-subscription-dial-out.md (hunk 1 above)
Comment (ruizoscar) [reply Teams/audio-conferencing-subscription-dial-out.md]: Thank you :)!

**Comment by ruizoscar** [comment]
@nenonix, it seems we're good with all of the comments, is there something I need to do to move this item forward?

**Comment by nenonix** [comment]
@ruizoscar if you see an option to "resolve" then please do so, otherwise the process continues as normal.

**Comment by ruizoscar** [comment]
I just clicked on every button I saw with the word "Resolve" 😄

**Comment by nenonix** [comment]
Thanks. Now it is over to the maintainers to merge your PR.

**Comment by LolaJacobsen** [comment]
@tonysmit @CarolynRowe OK to merge?

**Pull request merged**